Enable snapping of windows by default.

This patch originally by Thierry Reding, later modified/updated by Sean Finney
Not submitted to upstream.

Index: compiz/metadata/wobbly.xml.in
===================================================================
--- compiz.orig/metadata/wobbly.xml.in	2007-08-18 23:30:00.000000000 -0700
+++ compiz/metadata/wobbly.xml.in	2007-08-18 23:30:14.000000000 -0700
@@ -21,7 +21,7 @@
 	    <option name="snap_inverted" type="bool">
 		<_short>Snap Inverted</_short>
 		<_long>Inverted window snapping</_long>
-		<default>false</default>
+		<default>true</default>
 	    </option>
 	    <option name="shiver" type="action">
 		<_short>Shiver</_short>
